solution-base/build.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+set -eu
+
+PWD=$(pwd)
+BUILD_ROOT=${PWD}/_build
+ISO_ROOT=${BUILD_ROOT}/root
+IMAGES_ROOT=${ISO_ROOT}/images
+
+PRODUCT_NAME=Zenko-Base
+PRODUCT_LOWERNAME=zenko-base
+BUILD_TIMESTAMP=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+BUILD_HOST=$(hostname)
+
+VERSION_SHORT=$(git describe --abbrev=0)
+VERSION_FULL=${VERSION_SHORT}-dev
+GIT_REVISION=$(git describe --long --always --tags --dirty)
+ISO=${BUILD_ROOT}/${PRODUCT_LOWERNAME}-${VERSION_FULL}.iso
+
+DOCKER=docker
+DOCKER_OPTS=
+DOCKER_SOCKET=${DOCKER_SOCKET:-unix:///var/run/docker.sock}
+HARDLINK=hardlink
+SKOPEO=skopeo
+SKOPEO_OPTS="--override-os linux --insecure-policy"
+OPERATOR_TAG=$(grep kubedb/operator: deps.txt | awk -F ':' '{print $2}')
+SOLUTION_REGISTRY=metalk8s-registry-from-config.invalid/${PRODUCT_LOWERNAME}-${VERSION_FULL}
+
+KUBEDB_SCRIPT_BRANCH_TAG=89fab34cf2f5d9e0bcc3c2d5b0f0599f94ff0dca
+
+SOLUTION_ENV='SOLUTION_ENV'
+
+export KUBEDB_OPERATOR_TAG=${OPERATOR_TAG}
+export KUBEDB_NAMESPACE=${SOLUTION_ENV}
+export KUBEDB_SERVICE_ACCOUNT=kubedb-operator
+export KUBEDB_OPERATOR_NAME=operator
+export KUBEDB_DOCKER_REGISTRY=${SOLUTION_REGISTRY}
+export KUBEDB_PRIORITY_CLASS=system-cluster-critical
+
+KAFKA_NAME='kafka-operator'
+KAFKA_NAMESPACE=${SOLUTION_ENV}
+KAFKA_CERT_NAME='kafka-operator-serving-cert'
+KAFKA_IMAGE_NAME="/kafka-operator"
+KAFKA_IMAGE_REPO_NAME="${SOLUTION_REGISTRY}${KAFKA_IMAGE_NAME}"
```

Comment:
nitpicking, I don't know if I like the `/` in the image name, I rather do it like that
```suggestion
KAFKA_IMAGE_NAME="kafka-operator"
KAFKA_IMAGE_REPO_NAME="${SOLUTION_REGISTRY}/${KAFKA_IMAGE_NAME}"
```

But, we don't need to take this comment into account, you can skip it :smile: 

I'm also starting to think there's a reason behind this :thinking: